.scala-steward.conf

```diff
@@ -0,0 +1,12 @@
+updates.ignore = [
+  { groupId = "com.google.protobuf", artifactId = "protobuf-java" },
+  { groupId = "org.scalatest", artifactId = "scalatest" },
+  { groupId = "com.typesafe", artifactId = "config" },
+  { groupId = "com.typesafe", artifactId = "ssl-config-core" },
+  { groupId = "com.typesafe", artifactId = "sbt-mima-plugin" },
+  { groupId = "com.typesafe.sbt", artifactId = "sbt-multi-jvm" },
```

Comment:
why wouldn't we use the latest multi-jvm, if we are maintaining it?

Reply:
I thought with some of ours we want to manually.